internal/oci/container_linux.go

```diff
@@ -104,3 +107,25 @@ func getPidStatDataFromFile(file string) (string, string, error) { //nolint:gocr
 
 	return string(statFields[stateFieldIndex]), string(statFields[startTimeFieldIndex]), nil
 }
+
+// SetRuntimeUser sets the runtime user for the container.
+func (c *Container) SetRuntimeUser(spec *specs.Spec) {
+	runtimeSpec := spec
```

Comment:
Why not name the function argument `runtimeSpec`?

Reply:
Fixed